docs/workflows/api-token.md

```diff
@@ -95,6 +95,11 @@ In the Beyond Identity Admin Console under the "Apps" tab, select the "Beyond Id
 
 ## Revoking Access Tokens
 
+Depending on what information is available to you, you may choose to use one of the ways to revoke an access token.
+It's recommended to use RFC7009 revoke endpoint if you can.
```

Comment:
Why is this recommended over other options? Maybe a quick sentence explaining that or the main difference?

Reply:
You're right, this is not a good explanation. I updated the documentation to say "if you have raw token, use RFC-7009 endpoint. if you have token id, use management api". Would this make more sense?